packages/cli/src/controllers/__tests__/auth.controller.test.ts

```diff
@@ -98,5 +103,78 @@ describe('AuthController', () => {
 				withScopes: true,
 			});
 		});
+
+		it('should not login members if "OIDC" is the authentication method', async () => {
```

Comment:
Should we make this clear that it's email login?
Something like this?

```suggestion
		it('should not allow email login if "OIDC" is the authentication method', async () => {
```